basilisk/src/autotest.sh

```
@@ -13,27 +14,36 @@ EOF
-		    if make -q -f $makefile $basename.tst; then
-			echo $d/$f is up to date
+		    if test -f $basename/fail.tst; then
+			cp -a $basename/fail.tst $basename.s
+		    fi
+		    if make -q -f $makefile $basename.s; then
+			if test -f $basename/fail.tst; then
+			    rm -f $basename.s
+			fi
+			# echo $d/$f is up to date
 		    else
+			while test `ssh $SANDBOX tsp | grep -E '(running|queued)' | wc -l` -ge 10; do
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)